commcare_translations.py

```diff
@@ -54,5 +54,7 @@ def dumps(dct):
             val = u'\u00A0'
         # get rid of newlines
         val = val.replace('\n', '\\n')
+        # escape starting # character
+        val = re.sub('^#', '\#', val)
```

Comment:
Hey,

Will pride added escaping, I think our tests have examples of it, but don't remember off the top of my head how it works.

Comments can come anywhere in the line, so #'s need to be escaped everywhere they exist.